test/groovy/CloudFoundryDeployTest.groovy

```diff
@@ -91,10 +91,11 @@ class CloudFoundryDeployTest extends BasePiperTest {
         nullScript.commonPipelineEnvironment.configuration = [
             general: [
                 camSystemRole: 'testRole',
-                cfCredentialsId: 'myCreds'
+                cfCredentialsId: 'test_cfCredentialsId'
```

Comment:
Why do we need to change this?

Reply:
The `test_cfCredentialsId` are added as valid credentials in init phase. Previously, this test was testing an error which was thrown when no deployTool was set, thus it didn't matter which credentials are selected. 
Now, it is tested that the automatic inference of the deployTool works, and thus the whole deployment phase should be executed and valid credentials are required.